sdk/src/utils.ts

```diff
@@ -877,37 +883,47 @@ export const dailyFactorPool = (tokenXamount: BN, volume: number, feeTier: FeeTi
   return (volume * fee) / tokenXamount.toNumber()
 }
 
-export const poolAPY = (params: ApyPoolParams): number => {
+export const poolAPY = (params: ApyPoolParams) => {
```

Comment:
It would look much better if params will be destructuried instead of such references `params.feeTier`